test/bigtest/tests/ledgers/LedgersList/LedgersList-test.js

```diff
@@ -15,14 +17,10 @@ describe('Ledgers list', () => {
   beforeEach(async function () {
     this.server.createList('ledger', LEDGERS_COUNT);
 
-    this.visit('/finance/ledger');
+    this.visit(LEDGERS_ROUTE);
     await ledgersList.whenLoaded();
   });
 
-  it('is no results message label present', () => {
```

Comment:
I'm not sure if we need to remove this test. Could it be fixed ? basically, in interactor should be something like this `isNoResultsMessageLabelPresent = isPresent('[class*=mclEmptyMessage---]');`

Reply:
I think this test is useless cause data is already fetched before it and no result message can't be present

Reply:
yes, in case we don't need to specify any filter to fetch results it could be deleted